### PR TITLE
党派プラグインは翻訳中されます！Resolves #162. Supercedes #97.

### LINKED ISSUE #162: Translations
We needs translations. And I'm making this issue so I can resolve it, yeah :D

### PR DESCRIPTION
**Pull request by EvelynSubarrow**
_Touha puraguin wa hon'yaku naka saremas(u)!_
_Factions plugin can into translation!_

### CODE REVIEW

File: src/main/java/com/massivecraft/factions/cmd/CmdAdmin.java [hunk 1]
```diff
@@ -65,8 +67,8 @@ public void perform() {
         // if target player is currently admin, demote and replace him
         if (fyou == admin) {
             targetFaction.promoteNewLeader();
-            msg("<i>You have demoted %s<i> from the position of faction admin.", fyou.describeTo(fme, true));
-            fyou.msg("<i>You have been demoted from the position of faction admin by %s<i>.", senderIsConsole ? "a server admin" : fme.describeTo(fyou, true));
+            msg(TL.COMMAND_ADMIN_DEMOTES.toString(), fyou.describeTo(fme, true));
+            fyou.msg(TL.COMMAND_ADMIN_DEMOTED.toString(), senderIsConsole ? TL.GENERIC_SERVERADMIN : fme.describeTo(fyou, true));
```
Comment (drtshock) [inline src/main/java/com/massivecraft/factions/cmd/CmdAdmin.java]: missed a toString here

File: src/main/java/com/massivecraft/factions/cmd/CmdHelp.java [hunk 2]
```diff
@@ -126,50 +127,51 @@ public void updateHelp() {
...
-        pageLines.add("");
-        pageLines.add(p.txt.parse("<i>Damage from enemies is reduced in your own territory."));
-        pageLines.add(p.txt.parse("<i>When you die you lose power. It is restored over time."));
-        pageLines.add(p.txt.parse("<i>The power of a faction is the sum of all member power."));
-        pageLines.add(p.txt.parse("<i>The power of a faction determines how much land it can hold."));
-        pageLines.add(p.txt.parse("<i>You can claim land from factions with too little power."));
+        pageLines.add(p.txt.parse(TL.COMMAND_HELP_RELATIONS_5.toString()));
+        pageLines.add(p.txt.parse(TL.COMMAND_HELP_RELATIONS_6.toString()));
+        pageLines.add(p.txt.parse(TL.COMMAND_HELP_RELATIONS_7.toString()));
+        			pageLines.add(TL.COMMAND_HELP_RELATIONS_8.toString());
```
Comment (drtshock) [inline src/main/java/com/massivecraft/factions/cmd/CmdHelp.java:140]: huh

File: src/main/java/com/massivecraft/factions/zcore/util/TL.java [hunk 3]
```diff
@@ -21,8 +21,412 @@
...
+	COMMAND_FWARP_TOWARP("to warp"),
+	COMMAND_FWARP_FORWARPING("for warping"),
+	COMMAND_FWARP_WARPS("Warps: "),
+	
+	COMMAND_HELP_404("<b>This page does not exist"),
+	COMMAND_HELP_NEXTCREATE("<i>Learn how to create a faction on the next page."),
+	COMMAND_HELP_INVITATIONS("command.help.invitations","<i>You might want to close it and use invitations:"),
+	COMMAND_HELP_HOME("<i>And don't forget to set your home:"),
+	COMMAND_HELP_BANK_1("<i>Your faction has a bank which is used to pay for certain"),
+    COMMAND_HELP_BANK_2("<i>things, so it will need to have money deposited into it."),
```
Comment (drtshock) [inline src/main/java/com/massivecraft/factions/zcore/util/TL.java:148]: weird indentations? I can just format it before I push.

Comment (EvelynSubarrow) [inline src/main/java/com/massivecraft/factions/zcore/util/TL.java:148]: Weird indentations==best indentations

**Comment by cnr** [comment]
Merged as https://github.com/drtshock/Factions/commit/1230803cfb0288de6070e981d21a24881f0d3ab6.

**Comment by drtshock** [comment]
Thanks a ton!

**Pull request closed without merging**